code/datums/communications.dm

```diff
@@ -125,12 +125,14 @@ datum/radio_frequency
 								device.receive_signal(signal, TRANSMISSION_RADIO, frequency)
 					else
 						device.receive_signal(signal, TRANSMISSION_RADIO, frequency)
+					LAGCHECK(LAG_REALTIME)
```

Comment:
should maybe check if device exists to avoid runtimes in the event that one is deleted mid ping since it's lag check'd